### PR TITLE
Fix some typos in markdown files

### PR DESCRIPTION
**Pull request by qmonmert**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: BLUEPRINTS.md [hunk 1]
```diff
@@ -43,6 +43,6 @@ For this reason try to _avoid exact version at blueprint dependencies_ and disab
 - priorities names are static constants at the generator class.
   `get initializing() {}` -> `get [Generator.INITIALIZING]() {}`
   Motivation can be found at https://github.com/jhipster/generator-jhipster/blob/main/rfcs/3-jhipster-rfc-unambiguous-priorities.md.
-- many property migration can be warned by enabling `jhipster7Migration` feature at the constructor.
+- much property migration can be warned by enabling `jhipster7Migration` feature at the constructor.
```
Comment (mshima) [inline BLUEPRINTS.md:46]: much looks weird to me, not a native speaker.

File: BLUEPRINTS.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # JHipster Blueprints
 
-Blueprints allows to add new features or change current features.
+Blueprints allows adding new features or change current features.
```
Comment (mraible) [inline BLUEPRINTS.md:3]:
```suggestion
Blueprints allow adding new features or changing current features.
```

File: BLUEPRINTS.md (hunk 1 above)
Comment (mraible) [reply BLUEPRINTS.md:46]: I think "many" sounds better, but I'm also trying to understand what the sentence should say. Are you trying to say many properties can be migrated automatically, or the deprecated ones can be printed to the console?

**Review by mraible — APPROVED** [review]

File: BLUEPRINTS.md (hunk 1 above)
Comment (mshima) [reply BLUEPRINTS.md:46]: Removed properties will work but printed to console.
Won't work with the flag disabled.

**Review by DanielFran — APPROVED** [review]

**Pull request merged**